ROADMAP.md

```diff
@@ -0,0 +1,171 @@
+# Babel Roadmap
+
+> Not everything is set in stone or has an issue for it! Looking to post these to get more people involved or integrate with other projects.
+
+## Ecosystem
+
+### Remake `compat-table` used in preset-env
+
+> https://github.com/kangax/compat-table could use a remake, ideally work with browser vendors on this
+    
+- There is also https://github.com/mdn/browser-compat-data
+- Also use data from test262?
+- Run tests against real browsers
+- have a data-only format
+- Need continued maintainence
+
+### Polyfill behavior
+
+> This is regarding https://github.com/babel/babel/tree/master/packages/babel-preset-env#usebuiltins-usage
+
+- Allow any substitute polyfill instead of `core-js`. You should be able to override anything (custom `Promise`, etc)
+- Make `"usage"` option the default after it is stable.
+
+### Build/publish workflow
+  - Guide on compiling/publishing ES2015+, .mjs, etc: https://twitter.com/philwalton/status/908082461799616512
+  - Support multi-build/folder outputs?
+
+### Codemods for TC39 Proposals
+
+> Lebab/others are already used to convert from ES5 -> ESNext, so incorporate it into Babel itself.
+
+- Refactor [Lebab](https://github.com/lebab/lebab/issues/138) as Babel transforms (can keep the cli since it's a separate tool)
+  - Usecase: ES3 -> ES6+ (on source code)
+  - Usecase: Remove usage of dropped proposals
```

Comment:
wouldnt this just require something like `--write` to have your files updated in place?

Reply:
Maybe just a standalone tool IMO

Reply:
Yeah, sure - encapsulating this in other tool will be much safer for the users. I meant this just as an example and to point out that this one is probably quite a low hanging fruit, beside creating a separate CLI tool actual algorithms etc are pretty much done.

Reply:
Kind of, you still have to run Babel via recast so that it retains the previous formatting which means just run the plugin via babel-codemod (or just run prettier right after which works too). This also requires that those tools are updated to the latest syntax's which they aren't since they are not sync'd with this repo.

Also it maybe easier to write a separate transform because the output from Babel may not be what you are looking for in the source code (or you need to output the loose mode version).

Reply:
I mean to say that `lebab` already is the standalone + cli tool. It's basically the same thing as babel but with the printer being recast instead of babel-generator. This is also related to the idea of combining recast/prettier/babel-generator somehow

Reply:
Yeah it can stay it's own repo but if it uses `@babel/core` it can stay a cli tool and can be versioned separately. cc @nene @mohebifar on this again for more thoughts?